src/Integration/LocalServices/ILocalService.cs

```diff
@@ -10,7 +10,7 @@ namespace SonarLint.VisualStudio.Integration
     /// <summary>
     /// Marker for local service
     /// </summary>
-    internal interface ILocalService
+    public interface ILocalService
```

Comment:
It would be worth expanding the doc comment slightly, particularly now the interface is public - it's not clear what a "local service" is or what it is for.